DatadogCore/Tests/DatadogObjc/DDNSURLSessionDelegateTests.swift

```diff
@@ -7,6 +7,7 @@
 import XCTest
 import TestUtilities
 import DatadogInternal
+import DatadogRUM
```

Comment:
Do we need this? RUM and URLSessionDelegate seem unrelated

Reply:
true - not required. I added to test things before making objc interfaces. Good call. Removed.